src/home/components/DefaultHomeCard.js

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright © 2021-2023 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+import React from 'react';
+
+import {
+  Box,
+  Button,
+  Divider,
+  Link,
+  Paper,
+  Stack,
+  Typography,
+} from '@mui/material';
+
+import CardActionRouterLink from 'common/components/CardActionRouterLink';
+import RouterLink from 'common/components/RouterLink';
+
+import HomeCard from './HomeCard';
+
+const DefaultHomeCard = ({
+  item,
+  title,
+  link,
+  url,
+  pictureUrl,
+  image,
+  alt,
+  children,
+  showActionsAsButtons = false,
+  backgroundColor = '#F5F7FC',
+}) => (
+  <HomeCard
+    aria-labelledby={`${item}-default-title`}
+    sx={{ flexDirection: 'column' }}
+  >
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        padding: 2,
+      }}
+    >
+      <Typography
+        id={`${item}-default-title`}
+        variant="h2"
+        sx={{ pt: 0, pb: 2, textTransform: 'capitalize' }}
+      >
+        {title}
+      </Typography>
+      <Box height={128}>
+        <Stack
+          direction="row"
+          spacing={4}
+          sx={{
+            position: 'absolute',
+            backgroundColor: { backgroundColor },
+            width: 511,
+          }}
+        >
+          <Link component={RouterLink} to={pictureUrl}>
+            <Paper
+              variant="outlined"
+              component="img"
+              src={image}
+              alt={alt}
+              href={url}
+              height={128}
+            ></Paper>
+          </Link>
+          {children}
+          {/* <Typography variant="body1" sx={{ pt: 2.5 }}>
+            {content}
+          </Typography> */}
+        </Stack>
+      </Box>
+    </Box>
+    {showActionsAsButtons ? (
+      <Box
+        spacing={2}
+        textAlign="center"
+        sx={{ backgroundColor: '#FFF', pb: 1.5 }}
+      >
+        <Button
+          variant="contained"
+          size="medium"
+          sx={{ color: 'white' }}
+          href={url}
+        >
+          {link}
+        </Button>
+      </Box>
```

Comment:
It seems in here you can use more specific components for what you need, like CardActions and RouterButton, also it seems that you should not need the background since the background of the parent should work from what I see in Figma
```suggestion
      <CardActions
        textAlign="center"
        sx={{ pb: 1.5 }}
      >
        <RouterButton
          variant="contained"
          size="medium"
          sx={{ color: 'white' }}
          to={url}
        >
          {link}
        </RouterButton>
      </CardActions>
```